packages/affine/shared/src/services/theme-service.ts

```diff
@@ -218,6 +218,7 @@ const toolbarColorKeys: Array<keyof AffineCssVariables> = [
   '--affine-text-primary-color',
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +213]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->For this color button CSS style, I think we can use `--affine-background-primary-color` instead of `--affine-white`.

Include `--affine-white` has a little bit of risk here, as it is so common. I suggest only including `toolbar` UI CSS variables, like background, icon, and divider.<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/8685?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->